src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -539,6 +540,19 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter) extend
 
   def lineCommand(what: String): Result = editCommand(what, None)
 
+  def completionsCommand(what: String): Result = {
```

Comment:
Would it be worth generalizing this to optionally provide the string of code that follows the cursor?

```
scala> object Foo { def bar = { fo<TAB> }; def foo: Int = 42 }

scala> object Foo { def bar = { foo }; def foo: Int = 42 }
```

Reply:
That would require the cursor position. I would first like to do a version that is compatible with `sbt`'s completion command and then change both if this is wanted.